static/js/list_rendering.js

```diff
@@ -104,6 +104,7 @@ var list_render = (function () {
             // Needs to be enough to exceed the max height, so that a
             // scrollable area is created.
             init: function () {
+                ui.update_scrollbar($(".subscriber_list_container"));
```

Comment:
did a quick check, calling `ui.update_scrollbar($("#subscription_overlay .subscriber_list_container"));` after the list renders in `show_subscription_settings` doesn't result in the correct behaviour. will wait on Brock's reply before changing this!